test/test_basic.py

```diff
@@ -100,6 +100,9 @@ def test_group_merge(request):
     for vol in model.volumes:
         new_group.add_set(vol)
 
+    # using create_group should give same thing
+    assert model.create_group('mat:fuel') == new_group
```

Comment:
Should we also add a test for a new group?

Reply:
I'm a little confused here, do you mean a test for the presence of a new group?

Reply:
I just added a new test below for group creation when the group doesn't already exist.

Reply:
Ah, duh. Thanks @paulromano :)